.gitignore

```diff
@@ -25,4 +25,6 @@ tlvf.log
 #klockwork
 kw_reports/*.tmp
 kw_reports/rdkb/*.tmp
-kw_reports/ugw/*.tmp
\ No newline at end of file
+kw_reports/ugw/*.tmp
+
+.vscode/ipch
```

Comment:
Again, I don't think this is appropriate.

Reply:
What do you mean?